www/include/monitoring/objectDetails/common-func.php

```diff
@@ -45,31 +54,40 @@ function hidePasswordInCommand($command_name, $host_id, $service_id)
 
     $pearDBStorage = new CentreonDB('centstorage');
 
-    /* Get executed command lines */
+    /* Get command line with macro */
+    $query_command_line = "SELECT command_line FROM command WHERE command_name = '" .
+    $pearDB->escape($command_name) . "'";
```

Comment:
binding the values prevent from SQL injection

Reply:
Yes but all parameters are from PHP and not user ;)